src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/DisableRuntimeMarshalling.cs

```diff
@@ -127,32 +138,237 @@ internal sealed class DisableRuntimeMarshallingAnalyzer : DiagnosticAnalyzer
         public override void Initialize(AnalysisContext context)
         {
             context.EnableConcurrentExecution();
-            context.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.None);
+            // Report diagnostics in generated code to enable this analyzer to catch usages of
+            // delegates with non-blittable parameters that do not have [UnmanagedFunctionPointer] in source-generated interop.
+            context.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.ReportDiagnostics);
```

Comment:
Note that with the new argument value passed here, you will not get callbacks to symbols completely defined in generated code. `GeneratedCodeAnalysisFlags.ReportDiagnostics` just means that I only want to analyze and get callbacks for non-generated code, but my analysis might report diagnostics in generated code (say location in an auto-generated partial definition counterpart for a non-generated partial definition). If this is what you want, than this change is fine. However, if you wish to both get callbacks for all generated code + report diagnostics in generated code, you should use `GeneratedCodeAnalysisFlags.Analyze | GeneratedCodeAnalysisFlags.ReportDiagnostics` as the argument.